src/test/kotlin/io/libp2p/pubsub/gossip/Eth2GossipParams.kt

```diff
@@ -0,0 +1,96 @@
+package io.libp2p.pubsub.gossip
+
+import io.libp2p.etc.types.minutes
+import io.libp2p.etc.types.seconds
+import io.libp2p.etc.types.times
+
+// Spec constants
+val SlotDuration = 12.seconds
+val SlotsPerEpoch = 32
+val EpochDuration = SlotDuration * SlotsPerEpoch
+val BlocksTopic = "/eth2/00000000/beacon_block/ssz_snappy"
+val AggrAttestTopic = "/eth2/00000000/beacon_aggregate_and_proof/ssz_snappy"
+val AttestTopicPrefix = "/eth2/00000000/beacon_attestation_"
+val AttestTopicSuffix = "/ssz_snappy"
+
+val Eth2DefaultGossipParams = GossipParams(
```

Comment:
I couldn't find the source for these params to confirm they are all set correctly.  But as I understand it at this point they're only used for the test - we'd have to specifically chose to use them on the Teku side before scoring is activated right?

We just need to be cautious about how we enable peer scoring - probably should have a CLI option to toggle it for a while so we can test it really carefully in real world conditions.

Reply:
The source was some draft (I created this file a couple of months ago) from the group investigating Gossip params for Eth2. 
Right this is just for testing purposes for now and the precise numbers are not too important here. 
Yes, when enabling scoring in Teku we would need to revisit them for sure.